js/src/carousel.js

```diff
@@ -390,15 +391,20 @@ class Carousel {
 
   _setActiveIndicatorElement(element) {
     if (this._indicatorsElement) {
-      const indicators = [].slice.call(this._indicatorsElement.querySelectorAll(SELECTOR_ACTIVE))
-      $(indicators).removeClass(CLASS_NAME_ACTIVE)
+      const activeIndicator = this._indicatorsElement.querySelector(SELECTOR_ACTIVE)
+      $(activeIndicator).removeClass(CLASS_NAME_ACTIVE)
+      $(activeIndicator).removeAttr('aria-current')
 
-      const nextIndicator = this._indicatorsElement.children[
-        this._getItemIndex(element)
-      ]
+      const indicators = this._indicatorsElement.querySelectorAll(SELECTOR_INDICATOR)
 
-      if (nextIndicator) {
-        $(nextIndicator).addClass(CLASS_NAME_ACTIVE)
+      Number.parseInt = Number.parseInt || parseInt // IE11 mini-polyfill
```

Comment:
Just use `parseInt` directly, no need for polyfills.

Reply:
fair enough